channels/rdpdr/client/rdpdr_main.c

```diff
@@ -1132,14 +1157,15 @@ UINT rdpdr_send(rdpdrPlugin* rdpdr, wStream* s)
 	UINT status;
 	rdpdrPlugin* plugin = (rdpdrPlugin*) rdpdr;
 
+	if (!rdpdr || !s)
```

Comment:
I've looked at the caller, and I don't think this check is necessary. s can't be NULL (as we expect the caller to have checked the result of Stream_New), and I can't imagine that caller would pass us a NULL rdpdrPlugin.